libs/react-spa/bff/src/lib/bff.utils.ts

```diff
@@ -37,12 +37,27 @@ export const createQueryStr = (params: Record<string, string>) => {
   return new URLSearchParams(params).toString()
 }
 
+type UserCheckFn = (oldUser: BffUser, newUser: BffUser) => boolean
+
 /**
  *  This method checks if the user has a new session
  */
-export const isNewSession = (oldUser: BffUser, newUser: BffUser) => {
+export const isNewSession: UserCheckFn = (oldUser, newUser) => {
   const oldSid = oldUser.profile.sid
   const newSid = newUser.profile.sid
 
-  return oldSid && newSid && oldSid !== newSid
+  return !!(oldSid && newSid && oldSid !== newSid)
```

Comment:
This might be the issue, if `oldSid` or `newSid` is undefined or null this will return false. Is `oldSid` incorrectly empty making this check becoming false?

Reply:
No the session is the same